wpilibj/src/main/java/edu/wpi/first/wpilibj/simulation/DCMotorSim.java

```diff
@@ -24,7 +24,7 @@ public class DCMotorSim extends LinearSystemSim<N2, N1, N2> {
   /**
    * Creates a simulated DC motor mechanism.
    *
-   * @param plant The linear system representing the DC motor. This system can be created with
+   * @param plant The linear system representing the DC motor. The system must use radians as the distance unit.
```

Comment:
This doesn't match C++.

Reply:
I changed it to mention both overloads of LinearSystemId.createDCMotorSim, similar to the C++ docs, but I'm retaining the radians note for the `(double kV, double kA)` variant since I think that's an important note to have.

Reply:
C++ should have that note too because LinearSystem uses double.